templates/kobo-docker/docker-compose.backend.secondary.override.yml.tpl

```diff
@@ -3,13 +3,19 @@ version: '2.2'
 
 services:
   postgres:
-    extends:
-      file: docker-compose.backend.template.yml
-      service: postgres
```

Comment:
Because it was a bad copy/paste. `docker-compose.backend.secondary.override.yml` cannot be used without `docker-compose.backend.secondary.yml`. 
`docker-compose.backend.secondary.yml` is the file which extends the template (whereas `docker-compose.backend.secondary.override.yml` overrides sections declared in `docker-compose.backend.secondary.yml`)

Is it clear?